File: app/src/main/java/com/battlelancer/seriesguide/extensions/WebSearchExtensionReceiver.java
package com.battlelancer.seriesguide.extensions;

import com.battlelancer.seriesguide.SgApp;
import com.battlelancer.seriesguide.api.SeriesGuideExtension;
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver;

public class WebSearchExtensionReceiver extends SeriesGuideExtensionReceiver {

    @Override
    protected int getJobId() {
        return SgApp.JOB_ID_EXTENSION_WEBSEARCH;
    }

    @Override
    protected Class<? extends SeriesGuideExtension> getExtensionClass() {
        return WebSearchExtension.class;
    }
}

File: app/src/main/java/com/battlelancer/seriesguide/extensions/WebSearchExtensionReceiver.kt
package com.battlelancer.seriesguide.extensions

import com.battlelancer.seriesguide.SgApp
import com.battlelancer.seriesguide.api.SeriesGuideExtension
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver

class WebSearchExtensionReceiver : SeriesGuideExtensionReceiver() {
    override fun getJobId(): Int = SgApp.JOB_ID_EXTENSION_WEBSEARCH

    override fun getExtensionClass(): Class<out SeriesGuideExtension> =
        WebSearchExtension::class.java
}